binary.js

```diff
@@ -0,0 +1,36 @@
+function getCurrentTime() {
+    const date_now = new Date();
+    const hours = date_now.getHours().toString().padStart(2, "0");
+    const minutes = date_now.getMinutes().toString().padStart(2, "0");
+    const seconds = date_now.getSeconds().toString().padStart(2, "0");
```

Comment:
```suggestion
    const dateNow = new Date();
    const hours = dateNow.getHours().toString().padStart(2, "0");
    const minutes = dateNow.getMinutes().toString().padStart(2, "0");
    const seconds = dateNow.getSeconds().toString().padStart(2, "0");
```

Using camelCase please.